README.md

```diff
@@ -22,78 +23,86 @@ Here's a recipe template.
 | Estimated IBU | 30.05 |
 | Estimated Color | ![#000000](https://placehold.it/15/000000/000000?text=+) `#000000` |
 
-### Yeast: Safale S-04
-
-### Fermentables
+### Yeast/Bacteria
+| Type | Name |
+|---|---|
+| Yeast | Safale S-04 |
+| Bacteria | OYL-605 |
 
+### Mash Bill
 | Amount | Fermentable | L° | Bill % |
 |---|---|---|---|
-
-<!-- | 8 lb | Maris Otter | 2.3 | 69.6% |
+<!--
+| 8 lb | Maris Otter | 2.3 | 69.6% |
 | 2 lb | Light Munich | 8 | 17.4% |
 | 12 oz | Caramel / Crystal 80L | 80 | 6.5% |
-| 12 oz | Special Roast | 50 | 6.5% | -->
-
-### Adjuncts
-
-| Amount | Ingredient | Bill % |
-|---|---|---|
+| 12 oz | Special Roast | 50 | 6.5% |
+-->
 
-<!-- OR -->
-<!-- No non-fermentable adjuncts added to this recipe. -->
-
-### Hop Schedule
-
-| Amount | Variety | Type | AA | Use | Time | IBU | Bill % |
-|---|---|---|---|---|---|---|---|
-
-<!-- | 1 oz | Northern Brewer | Pellet | 9.3 | Boil | 30 min | 24.02 | 33.3% |
-| 1 oz | Styrian Goldings | Pellet | 5.5 | Boil | 10 min | 6.7 | 33.3% |
-| 1 oz | Fuggles | Pellet | 4.5 | Boil | 5 min | 3.01 | 33.3% | -->
-
-### Water Profile
+### Hops
+| Amount | Variety | AA | Bill % |
```

Comment:
I like that you split the schedule into the steps section so it eliminates duplicate data and offers better context. I do think that pellet/flower/other is still relevant information when purchasing though.

Reply:
I assumed it shouldn't make a difference but after some research, it does.